src/crab/splitdbm/split_dbm.cpp

```
@@ -6,10 +6,39 @@
 
 namespace splitdbm {
 
-PotentialFunction SplitDBM::pot_func(const std::vector<Weight>& p) {
-    return [&p](const VertId v) -> Weight { return p[v]; };
+namespace {
+
+// Compute deferred relations: bounds from bound_source applied to relations from rel_source.
+Graph compute_deferred(const ReadableGraph auto& bound_source, ReadableGraph auto& rel_source, const size_t sz) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)